package-lock.json

```diff
@@ -0,0 +1,1205 @@
+{
```

Comment:
I'd rather not commit this `package-lock.json` file — my understanding is it [can't be published](https://docs.npmjs.com/files/package-lock.json) so users won't see it, which makes the `npm install -g` that our Travis tests do install a different set of dependencies than end users.

(Admittedly, we shouldn't be committing `yarn.lock` either)